src/types/index.d.ts

```diff
@@ -5,5 +5,3 @@ export type TestSeatingChart = (SeatingChart | EventManager | SeatsioDesigner) &
     props?: any,
     state?: string
 }
-
-export {}
```

Comment:
I don't think this export really serves any purpose in the context of this file. Sometimes an empty export is necessary for certain files to be included in a build step, but these types are only ever used in the tests, so it shouldn't be included in the build step anyway. If I'm wrong and there is a specific purpose for it, I'm happy to add it back though!